### PR TITLE
Filter visibility on/off (2nd try ;) )

### PR DESCRIPTION
**Pull request by DachsbauTV**
Thanks for your patience with me :)

I've done a new fork and a branch with the feature 
Everything is cleaned up now. 
Fingers crossed that it works.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -388,6 +388,23 @@ instance.prototype.updateTransitionList = async function() {
 	self.init_feedbacks();
 }
 
+instance.prototype.updateFilterList = async function() {
+	var self = this;
+
+await self.obs.send('GetSourceFilters).then(data => {
```
Comment (bryce-seifert) [inline index.js:408]: Needs a closing ' after Filters.

We also need to send it a source name as part of the request https://github.com/Palakis/obs-websocket/blob/4.9.0/docs/generated/protocol.md#getsourcefilters

File: index.js [hunk 2]
```diff
@@ -1123,6 +1045,36 @@ instance.prototype.action = function(action) {
 				'scene-name': sceneName
 			});
 			break;
+		case 'toggle_filter_item':
+				let visible = true
+				let sceneName = action.options.scene && action.options.scene != "" ? action.options.scene : null
+
```
Comment (bryce-seifert) [inline index.js:1200]: I believe the above 2 variables are already used and defined in the toggle_scene_item action

File: index.js (hunk 1 above)
Comment (DachsbauTV) [reply index.js]: I've added it, not sure if it works that way though

File: index.js (hunk 2 above)
Comment (DachsbauTV) [reply index.js:1200]: Sorry i don't understand the problem here :/

**Comment by bryce-seifert** [comment]
@DachsbauTV I integrated filter support with https://github.com/bitfocus/companion-module-obs-studio/pull/96, so should be able to test in the latest betas. Closing this pull.

**Pull request closed without merging**